### DISCUSSION TITLE
How to reassign stereo assignments after sanitization

### DISCUSSION THREAD

**Discussion by kheyer** [rdkit/rdkit#6657] [Q&A]
RDKit version 2023.03.3

I'm running into an issue where `Chem.SanitizeMol` is removing stereo assignments, causing `rdMolDescriptors.CalcNumAtomStereoCenters` to throw the error `numStereoCenters called without stereo being assigned`.

What method do I need to use to add this information back in after sanitization?

Example:

```python
from rdkit import Chem
from rdkit.Chem import rdMolDescriptors

def test_calc_stereo(mol):
    try:
        result = rdMolDescriptors.CalcNumAtomStereoCenters(mol)
        print(result)
    except Exception as e:
        print(e)

smile = 'O[C@@H](N)C'

mol1 = Chem.MolFromSmiles(smile)

mol2 = Chem.MolFromSmiles(smile)
Chem.SanitizeMol(mol2)

test_calc_stereo(mol1)
>1

test_calc_stereo(mol2)
>numStereoCenters called without stereo being assigned
```

**Comment by bp-kelley** [comment]
Chem.MolFromSmiles already calls SanitizeMol.  While I think this is a bug, calling SanitizeMol more than once is not really recommended.
